R/initialization.R

```diff
@@ -4,82 +4,84 @@
 #' @param stan_data a list of data elements that will be passed to stan
 #' @param params a dataframe of parameter values that are passed to stan
 #' to specify the priors in the model
+#' @param stdev a numeric value indicating the standard deviation to sample
+#' from when initializing, particularly from a standard normal. Also acts as
+#' a multiplier on the prior standard deviation, to restrict the initial value
+#' to be sampled from the center of the prior. Default is 0.01
```

Comment:
I think this documentation needs rewording for clarity. If the goal of this function is to initialize each parameter near its prior mode, should say that explicitly, and also clarify that the distributional shapes specified below are the prior distribution shapes from the actual model. But it's not clear to me that that's exactly true in all cases.

Also, if the goal is to be less diffuse than the priors, why not provide a multiplicative factor for transforming the prior sds, rather than hard coding, i.e. something like

```R
my_scalar_param <- rnorm(1, params$my_param_prior_mean, scaling_factor * params$my_param_prior_sd)
```

Reply:
I think we could and maybe should do something like that, though of course some of them are not parameterized by their mean and sd or logmean and logsd (which is fin, just requires a bit more work in those instances)

I can try to rework this to make it more general, knowing that it will change the initializations slightly but that should be fine.